android/app-example/src/main/java/com/badoo/ribs/example/app/RootActivity.kt

```diff
@@ -29,6 +32,9 @@ class RootActivity : RibActivity() {
                 override fun ribCustomisation(): Directory = AppRibCustomisations
                 override fun switcherInput(): ObservableSource<Switcher.Input> = Observable.empty()
```

Comment:
Personally, I am against using val fields in the deps, creates a lot of problems if you have local vals with the same name

Reply:
I will also prefer to have functions, not values.
I don't think that it will be problems with local vals(you just shouldn't create such with the same names).

But we can vote about this.

Reply:
Let's keep functions